tox.ini

```diff
@@ -20,9 +20,12 @@ basepython =
     py34: python3.4
     py35: python3.5
     py36: python3.6
+    py37: python3.7
+
 
 [testenv:lint]
 basepython=python
 extras=lint
 commands=
     flake8 {toxinidir}/py_ecc
+    mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --no-strict-optional --check-untyped-defs --disallow-incomplete-defs --disallow-untyped-defs --disallow-any-generics -p p2p -p py_ecc
```

Comment:
just noting my copy/paste error, you can drop the `-p p2p` from this command.  Also, I'm going to advocate that this be updated to the stretch goals command since this repo should be as strictly typed as possible given its cryptographic nature.

```
mypy --strict --follow-imports=silent --ignore-missing-imports --no-strict-optional -p py_ecc
```

Reply:
Opps, my bad, didn't noticed that, correcting.